supervisor/supervisor.go

```diff
@@ -141,23 +142,25 @@ func (sv *Supervisor) getHyperPod(container string, spec *specs.Spec) (hp *Hyper
 		return nil, fmt.Errorf("The container %s is already existing", container)
 	}
 	for _, ns := range spec.Linux.Namespaces {
-		if ns.Path != "" {
-			if strings.Contains(ns.Path, "/") {
-				return nil, fmt.Errorf("Runv doesn't support path to namespace file, it supports containers name as shared namespaces only")
+		if len(ns.Path) > 0 {
+			pidexp := regexp.MustCompile(`/proc/(\d+)/ns/*`)
+			matches := pidexp.FindStringSubmatch(ns.Path)
+			if len(matches) != 2 {
+				return nil, fmt.Errorf("Can't find shared container with network ns path %s", ns.Path)
 			}
-			if ns.Type == "mount" {
-				// TODO support it!
-				return nil, fmt.Errorf("Runv doesn't support shared mount namespace currently")
-			}
```

Comment:
I'm not sure what problem it will bring if I remove this check, we have an internal K8S, it works fine with this for some time.